<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectGlobalSearchWidget</class>
 <widget class="QWidget" name="ProjectGlobalSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListView" name="resoultList">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchLine"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="replaceLine"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replaceSelectedButton">
       <property name="text">
        <string>Replace selected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replaceAllButton">
       <property name="text">
        <string>Replace all</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
